openapi/platform/platform.yaml

```diff
@@ -6776,6 +6776,65 @@ components:
                 x-stoplight:
                   id: yflmky9gif6ro
                 type: string
+            adminNotes:
+              type: string
+              x-stoplight:
+                id: tnzahuoq5fsyi
+              description: Administration notes for the account.
+            socialUrls:
```

Comment:
<img width="361" alt="API_Gateway___Stoplight" src="https://github.com/vendasta/api-gateway-docs/assets/50189991/f9c162f2-d35d-4a33-84d9-559314fca3c0">

Reply:
We generally only support business pages for social URLs. We should indicate that in the descriptions. Ideally we would document any validation rules, such as a regexp `pattern`, as well. If you can find that easily please add it. If not, I don't want to increase the scope of the story since this is outside the team's domain.